packages/backend/src/misc/fetch.ts

```diff
@@ -6,7 +6,7 @@ import fetch from 'node-fetch';
 import { HttpProxyAgent, HttpsProxyAgent } from 'hpagent';
 import config from '@/config/index.js';
 
-export async function getJson(url: string, accept = 'application/json, */*', timeout = 10000, headers?: Record<string, string>) {
+export async function getJson(url: string, accept = 'application/json, */*', timeout = 10000, headers?: Record<string, string>) : Record<string, any> {
```

Comment:
JSON is not only an object, but string, number, array, null can be placed at the top level, which may cause a mismatch with the type definition.

Reply:
I didn't check before but now I saw that all uses of `getJson` are cast to a more specific type anyway. Same applies to `signedGet`. Reverted the type fix.